glue/sales_val_flagging.py

```diff
@@ -219,6 +219,12 @@ def finish_flags(df, start_date, manual_update):
     # Discard pre-2014 data
     df = df[df["meta_sale_date"] >= start_date]
 
+    if sales_to_write_filter["column"]:
+        df = df[
+            df[sales_to_write_filter["column"]].isin(sales_to_write_filter["values"])
+        ]
+        print(df)
+
```

Comment:
This is where we implement the `sales_to_write_filter` element from the new yaml structure. If the yaml (yaml object is `inputs`) is empty, it will skip over the logic. If it detects anything it will leverage the column and values keys to make a filter on the data.